spec/models/base_organisation_spec.rb

```diff
@@ -56,5 +56,37 @@
       org.save
     end
   end
+  
+  describe 'friendly_id' do
+    # TODO: more tests on friendly_id methods?
```

Comment:
sure, but I think you misunderstood my question.  

Your comment says "# TODO: more tests on friendly_id methods?" and I'm responding to that saying, "do we need more tests?".  You've answered as if I asked "do we have too many tests?".  The question is "do we *need* more?" and if the answer is "no" then we can delete the comment right? :-)